src/main/java/com/homework/mateakademy/repo/MessageRepo.java

```diff
@@ -0,0 +1,12 @@
+package com.homework.mateakademy.repo;
+
+import com.homework.mateakademy.domain.Message;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+
+public interface MessageRepo extends CrudRepository<Message, Long> {
```

Comment:
better name MessageRepo  -> MessageRepository
and better add here annotation @ Repository